front/src/reducers/osrdconf/stdcmConf/stdcmConfReducers.spec.ts

```diff
@@ -71,20 +71,38 @@ describe('stdcmConfReducers', () => {
     expect(state).toEqual(stdcmConfInitialState);
   });
 
-  it('should handle updateStdcmStandardAllowance', () => {
-    const initialTimeStandardAllowance = testDataBuilder.buildTimeStandardAllowance(10);
-    const store = createStore({
-      standardStdcmAllowance: initialTimeStandardAllowance,
-    });
+  it('should handle margins update', () => {
```

Comment:
To fix your failing test

```suggestion
  describe('should handle margins update', () => {
```

Reply:
Done !